File: legacy/src/main/scala/extracells/integration/opencomputers/OpenComputers.scala
package extracells.integration.opencomputers

import appeng.api.AEApi
import extracells.integration.Integration
import extracells.item.ItemOCUpgrade
import li.cil.oc.api.Driver
object OpenComputers {
	
	def init(){
		Driver.add(new DriverFluidExportBus)
		Driver.add(new DriverFluidExportBus.Provider)
		Driver.add(new DriverFluidImportBus)
		Driver.add(new DriverFluidImportBus.Provider)
		Driver.add(new DriverOreDictExportBus)
		Driver.add(new DriverOreDictExportBus.Provider)
		Driver.add(new DriverFluidInterface)
		Driver.add(new DriverFluidInterface.Provider)
		Driver.add(ItemOCUpgrade.getInstance())
		Driver.add(ItemOCUpgrade.getInstance().Provider)
		AEApi.instance.registries.wireless.registerWirelessHandler(WirelessHandlerUpgradeAE)
		OCRecipes.loadRecipes()
		ExtraCellsPathProvider
	}

}

File: legacy/src/main/java/extracells/integration/opencomputers/OpenComputers.java
package extracells.integration.opencomputers;

import appeng.api.AEApi;
import extracells.item.ItemOCUpgrade;
import li.cil.oc.api.Driver;

public class OpenComputers {
    public static void init(){
        Driver.add(new DriverFluidExportBus());
        Driver.add(new DriverFluidExportBus.Provider());
        Driver.add(new DriverFluidImportBus());
        Driver.add(new DriverFluidImportBus.Provider());
        Driver.add(new DriverOreDictExportBus());
        Driver.add(new DriverOreDictExportBus.Provider());
        Driver.add(new DriverFluidInterface());
        Driver.add(new DriverFluidInterface.Provider());
        Driver.add(new ItemOCUpgrade());
        Driver.add(new ItemOCUpgrade.Provider());
        AEApi.instance().registries().wireless().registerWirelessHandler(new WirelessHandlerUpgradeAE());
        OCRecipes.loadRecipes();
        new ExtraCellsPathProvider();
    }
}
